### DISCUSSION TITLE
MA has millions of CIFS or JCIFS queries, Query A and Query AAAA

### DISCUSSION THREAD

**Discussion by rvandet** [music-assistant/support#2097] [Q&A] [answered]
My Pihole has millions of these lines:
query[A] jcifs0_1_5b.kpn from 192.168.2.14** Apr 8 08:29:52: cached jcifs0_1_5b.kpn is NXDOMAIN

I guess this is the album-art from Music Asistant that is being cached. Is this an Pihole, HA issue or an MA issue?

The memory of my pihole is full due to this process. Anyone got to solve this?

**Comment by OzGav** [comment]
Seems like a pi hole problem. Personally I use Adguard and would limit the log age if things were filling up

https://www.google.com/search?q=pihole+log+too+big

**Comment by rvandet** [comment]
I think so too, but MA is making al these CIFS calls since last week i upgraded. Was there an change the may have led to these calls?

**Comment by OzGav** [comment]
@marcelveldt ?

**Comment by marcelveldt** [comment, marked as answer]
Nope there have been no changes but I wonder why all these (local) queries end up on your pihole at all.
I think its pretty common that a CIFS mount will do lots of dns queries while the directory structure is traversed (which happens on every sync).
